core/src/Xios.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "include/Xios.hpp"
 
+#include <boost/algorithm/string.hpp>
 #include <boost/date_time/posix_time/posix_time.hpp>
 #include <boost/format.hpp>
 #include <boost/format/group.hpp>
```

Comment:
I figured it's okay to use boost for trimming strings, given that it's already used for other stuff.

Reply:
yeah, I would say so.

The documentation suggests installing all of boost and the `spack` package I made includes `algorithms` as part of the base `spack` recipe. So, I think it makes sense to use it.

@timspainNERSC do you have any objections?